### PR TITLE
added cvss_v2 into notes and occurrences for both v1 and v1beta1.

### PR DESCRIPTION
**Pull request by hanqiuzh**
Updates:
- added cvssv2 & cvssv3 into v1beta1 occurrence detail (naming & type matching grafeas v1's existing `cvssv3`).
- added cvss_v2 into v1 note.
- added cvssv2 into v1 occurrence.

Signed-off-by: Han Zhang <hanqiuzh@google.com>

### CODE REVIEW

File: proto/v1beta1/vulnerability.proto [hunk 1]
```diff
@@ -238,7 +238,13 @@ message Details {
 
   VexAssessment vex_assessment = 10;
 
-  // Next free ID is 11.
+  // The cvss v2 score for the vulnerability.
+  CVSS cvssv2 = 11;
```
Comment (wkozlik) [inline proto/v1beta1/vulnerability.proto:242]: Could you use snake case for field names? i.e. cvss_v2 and cvss_v3.

File: proto/v1/vulnerability.proto [hunk 2]
```diff
@@ -245,5 +248,8 @@ message VulnerabilityOccurrence {
   // Output only. CVSS version used to populate cvss_score and severity.
   grafeas.v1.CVSSVersion cvss_version = 11;
 
-  // Next free ID is 12.
+  // The cvss v2 score for the vulnerability.
+  CVSS cvssv2 = 12;
```
Comment (wkozlik) [inline proto/v1/vulnerability.proto:252]: Could you use snake case for field names? i.e. cvss_v2.

**Review by wkozlik — CHANGES_REQUESTED** [review]

File: proto/v1beta1/vulnerability.proto (hunk 1 above)
Comment (hanqiuzh) [reply proto/v1beta1/vulnerability.proto]: Thanks. Updated.

File: proto/v1/vulnerability.proto (hunk 2 above)
Comment (hanqiuzh) [reply proto/v1/vulnerability.proto]: Updated

**Review by wkozlik — APPROVED** [review]
LGTM

**Pull request merged**